sequila/sequila-core/benches/databio_benchmark.rs

```diff
@@ -0,0 +1,299 @@
+        Table {
+            id: 0,
+            name: String::from("chainRn4"),
+            non_flat: 6,
+            table_size: TableSize::Medium,
+        },
+        Table {
+            id: 1,
+            name: String::from("fBrain-DS14718"),
+            non_flat: 1,
+            table_size: TableSize::Small,
+        },
+        Table {
+            id: 2,
+            name: String::from("exons"),
+            non_flat: 2,
+            table_size: TableSize::Small,
+        },
+        Table {
+            id: 3,
+            name: String::from("chainOrnAna1"),
+            non_flat: 6,
+            table_size: TableSize::Medium,
+        },
+        Table {
+            id: 4,
+            name: String::from("chainVicPac2"),
+            non_flat: 8,
+            table_size: TableSize::Medium,
+        },
+        Table {
+            id: 5,
+            name: String::from("chainXenTro3Link"),
+            non_flat: 7,
+            table_size: TableSize::Large,
+        },
+        Table {
+            id: 6,
+            name: String::from("chainMonDom5Link"),
+            non_flat: 7,
+            table_size: TableSize::Large,
+        },
+        Table {
+            id: 7,
+            name: String::from("ex-anno"),
+            non_flat: 2,
+            table_size: TableSize::Small,
+        },
+        Table {
+            id: 8,
+            name: String::from("ex-rna"),
+            non_flat: 7,
+            table_size: TableSize::Medium,
+        },
+    ];
+
+    const QUERY: &str = r#"
+            SELECT
+                count(*)
+            FROM
+                s1 a, s2 b
+            WHERE
+                a.contig=b.contig
+            AND
+                a.pos_end>=b.pos_start
+            AND
+                a.pos_start<=b.pos_end
+        "#;
+
+    // Actutal test cases
+    let test_cases = vec![
+        TestCase {
+            s1: tables[0].clone(),
+            s2: tables[1].clone(),
+        },
+        TestCase {
+            s1: tables[0].clone(),
+            s2: tables[3].clone(),
+        },
+        TestCase {
+            s1: tables[0].clone(),
+            s2: tables[7].clone(),
+        },
+        TestCase {
+            s1: tables[0].clone(),
+            s2: tables[8].clone(),
+        },
+        TestCase {
+            s1: tables[7].clone(),
+            s2: tables[8].clone(),
+        },
+    ];
+    let algorithms = [
+        Some(Algorithm::Coitrees),
+        Some(Algorithm::IntervalTree),
+        Some(Algorithm::ArrayIntervalTree),
+        Some(Algorithm::AIList),
+    ];
+    let runtime = tokio::runtime::Builder::new_current_thread()
+        .build()
+        .unwrap();
+
+    let mut group = c.benchmark_group("Algorithms");
+    group.sample_size(10);
+    group.measurement_time(std::time::Duration::from_secs(7200));
+    group.significance_level(0.1);
+
+    for test in test_cases {
+        let s1_path = root_path.clone() + &test.s1.name + &String::from("/*.parquet");
```

Comment:
i think it's better to prefer `format!()` for string concatenations:
```rust
        let s1_path = format!("{}{}{}", root_path, test.s1.name, "/*.parquet");
```

Reply:
Fixed